src/components/Card/cardStyles.tsx

```diff
@@ -1,5 +1,4 @@
-import  {makeStyles} from '@mui/styles';
-
+import  { makeStyles } from '@mui/styles';
```

Comment:
```suggestion
import { makeStyles } from '@mui/styles';
```

why do you add extra space?